de.fhg.igd.equinox.test.app/src/de/fhg/igd/equinox/test/app/runner/TestRunner.java

```diff
@@ -99,6 +103,59 @@ public void execute() {
 		log.info("Test runner completed.");
 	}
 
+	/**
+	 * Run tests for the given test classes.
+	 * 
+	 * @param exec the test executor
+	 * @param testClasses the test classes
+	 */
+	private void runTests(TestExecutor exec, List<Class<?>> testClasses) {
+		if (config.getMethodNames().isEmpty()) {
+			// run all test classes and their tests
+			exec.executeTests(testClasses, failures);
+		}
+		else {
+			// determine methods for classes
+			
+			List<TestClassAndMethods> tests = new ArrayList<TestClassAndMethods>();
+			for (Class<?> testClass : testClasses) {
+				Collection<String> methods = findTestMethods(testClass);
+				if (methods != null && !methods.isEmpty()) {
+					tests.add(new TestClassAndMethods(testClass, methods));
+				}
```

Comment:
Log if none of the methods were found in a class?

Reply:
I guess depends on the use case if that is helpful.
I think for the (right now for me) main use case of running a specific method in a specific test class the feedback at the end on how many total test methods were run is sufficient.
If a use case would also be to run specific test methods and check for them in all test classes or all test classes of specific bundles, then it would be probably expected that many test classes don't have the specific method and this would just be noise.